README.rst

```diff
@@ -563,6 +563,22 @@ their hardware ID instead of their name.
 
 If you still want to reference devices by their name, set this to ``False``.
 
+``OPENWISP_CONTROLLER_DEVICE_VERBOSE_NAME``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+-------------+
+| **type**:    |  ``str``    |
++--------------+-------------+
+| **default**: | ``Device``  |
++--------------+-------------+
+
+When enabled, makes device verbose name configurable in the
+``Network Configuration administration``.
```

Comment:
@codesankalp this is setting is not something which can be enabled or disabled like a boolean flag. This configures another setting.  This should be something along the lines of 
```
Defines the verbose_name attribute of Device model, which is displayed on admin site. 
```